pkg/rngd/cmd/rngd/rng_linux_arm64.go

```diff
@@ -10,12 +10,12 @@ import (
 	"errors"
 )
 
-// No standard RNG on arm64
-
-func initRand() bool {
+// No standard RNG instruction on arm64
+func initDRNG(ctx *rng) bool {
```

Comment:
maybe we should just change this file to be compiled when  `!linux,!amd64` as this will then be fine if we add other platforms or someone else wants to reuse the code.

Reply:
I don't quite understand, if we use `!linux,!amd` in this file to prevent it compiling, then we will get `undefined initDRNG and readDRNG` error in 'linux+arm64' environment, so my alternative is: if we want to support other platforms, e.g. PowerPC, we can add a file like `rng_linux_ppc.go` to support RNG instruction case if that arch does have one.